### DISCUSSION TITLE
请问一下大家有什么办法转化stl文件变为stp文件吗？

### DISCUSSION THREAD

**Discussion by jianfeng233** [peng-zhihui/HelloWord-Keyboard#28] [Q&A]
想根据键盘的3D模型做一些其它的结构设计，用fusion360的网格功能生成实体很容易将电脑卡死，试了好几次都把软件卡崩溃了，请教一下大家有没有其它方式转换stl文件？

**Comment by Mehver** [comment]
没有公式化的转换方式，目前来看不可能完美转换的。网上有些这类付费工具，我猜测方案还是利用AI，就算真转了，还是得手调。个人建议是从头再工业建模一次。网上找原理图有时候还得照着PDF自己画一遍呢，习惯就好。
